Changelog.md

```diff
@@ -17,8 +17,9 @@ Compiler Features:
 Bugfixes:
  * Assembly output: Do not mix in/out jump annotations with arguments.
  * Code Generator: Annotate jump from calldata decoder to function as "jump in".
- * Type Checker: Properly detect different return types when overriding an external interface function with a public contract function.
  * Optimizer: Fix nondeterminism bug related to the boost version and constants representation. The bug only resulted in less optimal but still correct code because the generated routine is always verified to be correct.
+ * Type Checker: Properly detect different return types when overriding an external interface function with a public contract function.
+ * Type Checker: Disallow struct return types for getters of public state variables unless the new ABI encoder is active.
```

Comment:
The code seems to allow simple structs though.

Reply:
Yeah, need to think whether this can be rephrased, but it's hard. If the resulting type is only a struct, then it is broken up and the getter actually returns multiple values instead of a struct.